Task_11/Main.java

```diff
@@ -0,0 +1,57 @@
+package Movement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class, which consist of methods main(input point in the program), meansOfTransport, which create list of
+ * them and print, which output on the screen.
+ * @author Irina Bogatko
+ * @version 1.1
+ * @since 26.10.2016
+ */
+public class Main {
+
+    /**
+     * Output point in the program
+     * @param args command line params.
+     */
+    public static void main(String[] args) {
+        ArrayList<Checkpoint> points = new ArrayList<Checkpoint>();
+        GetCheckpoints getCheckpoints = new GetCheckpoints();
+        Distance distance = new Distance();
+        points = getCheckpoints.readCheckpoints();
+        ArrayList<Trip> allTrip = meansOfTransport();
+        for (Checkpoint point : points) {
+            for (Trip vehicle : allTrip) {
+                vehicle.nextCheckpoint(point, distance);

```

Comment:
I afraid it won't work - every vehicle modify the same distance by adding checkpoints to it so your distance will grow form vehicle to vehicle